src/main/java/com/amazon/ion/impl/_Private_IonReaderBuilder.java

```diff
@@ -352,4 +352,16 @@ public IonTextReader build(String ionText) {
         return makeReaderText(validateCatalog(), ionText, lstFactory);
     }
 
+    /**
+     * Creates a new {@link MacroAwareIonReader} over the given data.
+     * @param ionData the data to read.
+     * @return a new MacroAwareIonReader instance.
+     */
+    public MacroAwareIonReader buildMacroAware(byte[] ionData) {
```

Comment:
Note: this is "hidden" in `_Private_IonReaderBuilder` much like `newSystemReader` is hidden in `_Private_IonSystem`.